roles/aodh/meta/main.yml

```diff
@@ -9,13 +9,14 @@ dependencies:
       - inspec.enabled|bool
       - inspec.controls.aodh.enabled|bool
   - role: openstack-source
-    project_name: aodh
```

Comment:
We should not make this change. Currently all 3 approaches, openstack-distro, openstack-package and openstack-source are this way.